plugins/haproxy_logs.yaml

```diff
@@ -0,0 +1,112 @@
+version: 0.0.1
+title: HAProxy
+description: Log parser for HAProxy
+parameters:
+  - name: file_path
+    type: "[]string"
+    default:
+      - "/var/log/haproxy/haproxy.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    filelog:
+      include:
+        {{ range $fp := .file_path }}
+        - '{{ $fp }}'
+        {{end}}
+      start_at: {{ .start_at }}
+      operators:
+        - id: initial_router
+          type: router
+          routes:
+            - output: http_parser
+              expr: 'body matches "^\\w{3}\\s+\\d{1,2}\\s+\\d{2}:\\d{2}:\\d{2}\\s+[^\\s]+\\s+[^\\[]+\\[[^\\]]+\\]:(\\s)?[^:]+:[^\\s]+\\s+\\[[^\\]]+\\]\\s+[^\\s]+\\s+[^/]+/[^\\s]+[^/]+/[^/]+/[^/]+/[^/]+/[^\\s]+\\s+[^\\s]+\\s+[^\\s]+\\s+[^\\s]+\\s+[^\\s]+\\s+[\\w-]{4}\\s+[^/]+/[^/]+/[^/]+/[^/]+/[^\\s]+\\s+[^/]+/[^\\s]+ ({[\\w\\d[:ascii:]]+}\\s)?({[\\w\\d[:ascii:]]+}\\s)?\"\\S+ +[^ ]*( [^/]*/[^\\\"]*|[^\\\"]*)?"'
+              attributes:
+                frontend_type: http
+            - output: tcp_parser
+              expr: 'body matches "^\\w{3}\\s+\\d{1,2}\\s+\\d{2}:\\d{2}:\\d{2}\\s+[^\\s]+\\s+[^\\[]+\\[[^\\]]+\\]:(\\s)?[^:]+:[^\\s]+\\s+\\[[^\\]]+\\]\\s+[^\\s]+\\s+[^/]+/[^\\s]+"'
+              attributes:
+                frontend_type: tcp
+            - output: general_parser
+              expr: 'body matches "^\\w{3}\\s+\\d{1,2}\\s+\\d{2}:\\d{2}:\\d{2}\\s+[^\\s]+\\s+[^\\[]+\\[[^\\]]+\\]: \\[[^\\]]+\\]"'
+            - output: generic_parser
+              expr: 'body matches "^\\w{3}\\s+\\d{1,2}\\s+\\d{2}:\\d{2}:\\d{2}\\s+[^\\s]+\\s+[^\\[]+\\[[^\\]]+\\]:(\\s)?.*"'
```

Comment:
These output names are very similar. I'm guessing this is what we called it in the original. Any chance we have names that might be more descriptive or less similar?

Reply:
I renamed it and tried to add a bit more context on when it is used.